Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmail.cs

```diff
@@ -1036,53 +1036,53 @@ private void ExecutedActivitiesDetailsGenrator(LiteDbRunSet liteDbRunSet, HTMLRe
             return chartData;
         }
 
-        private static void GetExecGeneralDetails(LiteDbRunSet liteDbRunSet, HTMLReportConfiguration currentTemplate, StringBuilder fieldsNamesHTMLTableCells, StringBuilder fieldsValuesHTMLTableCells, StringBuilder fieldsNamesHTMLTableCellsm, StringBuilder fieldsValuesHTMLTableCellsm)
+        private static void GetExecGeneralDetails(LiteDbRunSet liteDbRunSet, HTMLReportConfiguration currentTemplate, StringBuilder fieldsNamesHTMLTableCellsRowOne, StringBuilder fieldsValuesHTMLTableCellsRowOne, StringBuilder fieldsNamesHTMLTableCellsRowTwo, StringBuilder fieldsValuesHTMLTableCellsRowTwo)
         {
             string tableColor = "<td bgcolor='#7f7989' style='color:#fff;padding:10px;border-right:1px solid #fff'>";
             string tableStyle = @"<td style='padding: 10px; border: 1px solid #dddddd'>";
+
+            var totalColumnCount = currentTemplate.EmailSummaryViewFieldsToSelect.Where(x => (x.IsSelected == true && x.FieldType == Ginger.Reports.FieldsType.Field.ToString())).Count(); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4663891)